Sources/KituraContracts/CodableQuery/QueryEncoder.swift

```diff
@@ -21,23 +21,60 @@ extension CharacterSet {
     static let customURLQueryAllowed = CharacterSet(charactersIn: "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-._~=:&")
 }
 
-/// Query Parameter Encoder
+/**
+ Query Parameter Encoder.
```

Comment:
We should have a much better description here!? This doesn't tell me what it does or why I'd use it. The usage example doesn't have a description to help me out either. Please can we have a real example snippet here that can be compiled and run with an example `query`. The same applies for QueryDecoder.